addons/dialogue_manager/views/main_view.gd

```diff
@@ -157,6 +161,7 @@ func _ready() -> void:
 	# Connect menu buttons
 	insert_button.get_popup().id_pressed.connect(_on_insert_button_menu_id_pressed)
 	translations_button.get_popup().id_pressed.connect(_on_translations_button_menu_id_pressed)
+	translations_button.about_to_popup.connect(update_translations_menu)
```

Comment:
Isn't this missing a `get_popup()`? Also, can you rename the handler to follow the conventions used for other signal handlers?